com.unity.render-pipelines.universal/Shaders/Utils/Universal2D.hlsl

```diff
@@ -26,8 +26,7 @@ Varyings vert(Attributes input)
 
 half4 frag(Varyings input) : SV_Target
 {
-    half2 uv = input.uv;
-    half4 texColor = SAMPLE_TEXTURE2D(_BaseMap, sampler_BaseMap, uv);
+    half4 texColor = SAMPLE_TEXTURE2D(_BaseMap, sampler_BaseMap, input.uv);
```

Comment:
This code still gives us a warning as SAMPLE_TEXTURE2D returns a float4.
So you either need to have texColor as float4 or do a cast.